types/v56/types.go

```diff
@@ -2342,6 +2358,50 @@ type QueryResultVMWProviderVdcRecordType struct {
 	MemoryOverheadMB        int64  `xml:"memoryOverheadMB,attr,omitempty"`
 }
 
+// QueryResultOrgVdcRecordType represents all VDC in organization
+// Type: QueryResultOrgVdcRecordType
+// Namespace: http://www.vmware.com/vcloud/v1.5
+// Since: 1.5
+type QueryResultOrgVdcRecordType struct {
```

Comment:
We would need a test for the usage of this type

Reply:
I added a method for usage this type and a test